.gitignore

```diff
@@ -19,10 +19,13 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-protoc-gen-elixir
+# Ignore package tarball (built via "mix hex.build").
+protobuf-*.tar
+
+# Temporary files for e.g. tests.
+/tmp/
 
-# QuickCheck artifacts
+# QuickCheck artifacts.
+protoc-gen-elixir
```

Comment:
This is not a QuickCheck artifact, is it? It's the protoc compiler plugin

Reply:
> This is not a QuickCheck artifact, is it? It's the protoc compiler plugin

I will put under `# Misc.`.